src/components/Timeline/TimelineNote.tsx

```diff
@@ -23,6 +83,9 @@ interface TimelineNoteProps {
   relatedId?: string;
   onNoteUpdate?: (id: string, message: string) => Promise<unknown>;
   onNoteUpdateLoading?: boolean;
+  dateNode?: React.ReactNode;
+  eventData?: any;
```

Comment:
issue: let's not use `any`, if this is some expected object it should have it's own type